README.org

```diff
@@ -398,6 +398,11 @@ example you can put the following in your configuration:
 (setq ob-async-no-async-languages-alist '("jupyter-python" "jupyter-julia"))
 #+END_SRC
 
+*** Issues with =ob-ipython=
```

Comment:
Thanks for this.

Can you run `M-x org-fill-paragraph` (bound to `M-q`) on the two paragraphs so that they are formatted like the rest of the document.

Reply:
Sure, np, here is it